### PR TITLE
Allow Global Voices and higher to pm /code

### PR DESCRIPTION
**Pull request by AustinXII**

### CODE REVIEW

**Comment by AustinXII** [comment]
@Zarel

**Comment by Zarel** [comment]
This code does not check for global voice or higher...?

File: server/chat-plugins/info.js [hunk 1]
```diff
@@ -2329,6 +2330,7 @@ const commands = {
 	},
 	codehelp: [
 		`!code [code] - Broadcasts code to a room. Accepts multi-line arguments. Requires: + % @ & # ~`,
+		`In order to use !code in pms you must be a global voice or higher`,
```
Comment (Zarel) [inline server/chat-plugins/info.js]:
```suggestion
		`In order to use !code in private messages you must be a global voice or higher`,
```

**Comment by AustinXII** [comment]
@Zarel there is a room check, since pms are a  global room you need to be global voice or higher. I did test this :P

**Comment by Zarel** [comment]
Oh! So this was already implemented, we just accidentally omitted the `"!code": true` part to make it work.

**Comment by AustinXII** [comment]
Well you can broadcast /code in normal chat rooms so I don’t know if that was the case but it definitely made for an quick and easy PR

**Pull request merged**